demo/demo.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'ao-demo-app',
-  template: '<ao-employee-grid-odata></ao-employee-grid-odata>'
+  selector: 'wnp-demo-app',
```

Comment:
Why would you change this to `wnp` instead of `ao` (angular odata) ?

Reply:
Your lint rules require it to be wnp... I figured the lint rules were right. Should I remove/disable the lint rule instead?

Reply:
Ah, I see, I copy-pasted from another project, my mistake. Please change tslint to `oa`.

Reply:
OK, I just did that. A couple more changes coming shortly.